src/sentry/tasks/digests.py

```diff
@@ -42,7 +43,7 @@ def schedule_digests():
     queue="digests.delivery",
     silo_mode=SiloMode.REGION,
 )
-def deliver_digest(key, schedule_timestamp=None):
+def deliver_digest(key, schedule_timestamp=None, notification_uuid: Optional[str] = None):
```

Comment:
You may want an interim PR to use kwargs and then swap to a explicit variable.

Reply:
I'm told that this may be a 3 way rollout:
(1) add new optional parameter
(2) start sending new parameter
(3) remove optional-ness